src/ConsoleContactAppManager/Program.cs

```diff
@@ -0,0 +1,67 @@
+﻿// <copyright file="Program.cs" company="PlaceholderCompany">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
+
+namespace ContactManagerApp
+{
+    /// <summary>
+    /// Program class has the options to perform and it recives the option from user.
+    /// </summary>
+    internal class Program
+    {
+        /// <summary>
+        /// Options has the different options for the user to perform.
+        /// </summary>
+        private enum Options
```

Comment:
You can move this enum to a separate file.

Reply:
Enum moved to program1.cs file.